src/config.py

```diff
@@ -4,6 +4,24 @@
 
 class Config(object):
     SECRET_KEY = os.environ.get('SECRET_KEY') or 'you-will-never-guess'
-    SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL') or \
-        'sqlite:///' + os.path.join(basedir, 'app.db')
     SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+
+class ProductionConfig(Config):
```

Comment:
SQLALCHEMY_DATABASE_URI in both staging and production should take the environment variable as well. Hard-coded address should not appear in the code.